_xtools

```diff
@@ -136,6 +136,7 @@ _xtools() {
 		xbuildbarf) _xtools_one_arch "@";;
```

Comment:
Unrelated to this PR, but is this a typo?

```suggestion
		xbuildbarf) _xtools_one_arch "$@";;
```

Reply:
yes, make a separate commit please.

Reply:
(doesn't make a difference as $@ is ignored anyway)